src/v/kafka/server/quota_manager.h

```diff
@@ -129,6 +131,7 @@ class quota_manager : public ss::peering_sharded_service<quota_manager> {
       clock::time_point now,
       quota_mutation_callback_t cb);
     ss::future<> add_quota_id(std::string_view quota_id, clock::time_point now);
+    void update_client_quotas();
```

Comment:
nit: move to 06fcf7d9b0d63335dd6d726bf73713171522a58d I think

Reply:
Done